server/mailrepository/mailrepository-api/src/test/java/org/apache/james/mailrepository/MailRepositoryTest.java

```diff
@@ -0,0 +1,125 @@
+package org.apache.james.mailrepository;
```

Comment:
Missing license. You should solve this problem!

Reply:
Maybe a checkstyle?

Reply:
Yes, that's something we can do with checkstyle (http://checkstyle.sourceforge.net/config_header.html#Header). Do you want to handle such a PR ?

Reply:
fixed